framework/assets/yii.gridView.js

```diff
@@ -154,17 +183,20 @@
             if (gridData[id] === undefined) {
                 gridData[id] = {};
             }
+            if (gridEventHandlers[id] === undefined) {
+                gridEventHandlers[id] = {};
+            }
```

Comment:
should this check be moved to `initEventHandler`? sounds logical to me as it is done always before calling `initEventHandler`.